dockerfiles/components/clarinet.dockerfile

```diff
@@ -2,6 +2,8 @@ FROM debian:bookworm-slim
 
 RUN apt update && apt install -y libssl-dev
 
+RUN rustup update 1.71.0 && rustup default 1.71.0
```

Comment:
This file is not used _yet_
It was introduced in #1104 and rightfully used v1.67.0, reflecting what was set in rust-toolchain at the time